chapters/14-team-package-dev.Rmd

```diff
@@ -1,14 +1,93 @@
+- Setting up two-factor authentication
+    - Text: What is it and why to use it.
+    - Text: In general 2FA is highly highly encouraged, but on team-based projects,
+    its even more so important, especially for repo owners and organization "owners".
+    - Exercise: Set up 2FA for your GitHub account.
+- Branch protection
+    - Text: Used to assign a specific person or people to be responsible for specific
+    branches, to provide a level of accountability and prevent unreviewed content
+    from being added to package.
+    - Text: Generally, depending on skill of member in git/github, best give
+    more team members access to modifying branch than to have less. But ultimately
+    it is a safety and accountability feature that needs to be discussed with team.
+    - Exercise: 
+        - Solo: In the Organization repo, set up branch protection for the main branch.
+        - Team:
+            - In the organization repo, set up branch protection for the main branch
+            and assign the owners as the only ones who can push to the branch.
+            FIXME: Make sure this is the correct terminology.
+- Relying on and using code of conduct, contributing guide, governance to build
+culture and standards
+    - Text: An implicit team culture will naturally evolve as people work together,
+    but having purely implicit cultural norms makes it difficult for newcomers
+    and for dealing with problems that come from misunderstanding of these cultural
+    practices. Much of this comes from the code of conduct and contributing guidelines,
+    but there are other aspects that can be team-, country-, etc specific.
+    - Exercise:
+        - (Solo/team?): Brainstorm come explicit expected behaviours, norms, 
+        practices that you would like in a team environment. Write this text
+        down and put it in the (FIXME: where? governance? conduct?)
+    - Text: Consider how a team might handle, prevent/limit, and resolving
+    conflict and personality differences. These often are team-specific.
+    - Exercise:
+        - (Solo/team?): Brainstorm ideas for what things to write about and include
+        in an explicit document about how these are handled.
+- Running meetings
+    - Where would text be about this? Contributing? Governance?
+    - Text: Running effective meetings is a key skill to working as a team.
+    - Text: List some best practices.
+    - Exercise: 
+        - Consider past meetings you've had. Think of the worse and best meetings.
+        What made a good meeting? What made a bad one?
+        - Team: Discuss and brainstorm as a team some practices you agree with 
+        and want to implement in your meetings.
+- Governance (team focused)
+    - Text: Governance is being explicit about the structure of the team, who does
+    what and is responsible for what, how decisions are made, etc.
+    - Exercise:
+        - Solo: ? FIXME: add something here?
```

Comment:
How about an exercise in thinking of tasks to delegate:

> Make a list of project administration or upkeep tasks (such as labeling and assigning user-created GitHub Issues, initiating meetings or check-ins, or fixing bugs). For each task, describe what changes to your project documentation (if any) would be necessary to enable someone else to do that task. What information and expertise would they need? How familiar would they need to be with the project?

This has overlap with README design and issue-creating in chapters 10 and 12, but here it could be a more pointed thought exercise specifically about delegation.

Reply:
Nice idea!